test/functionalTest/harnessFunctions.sh

```diff
@@ -428,6 +428,16 @@ function localBrokerStart()
 
   ps=$(ps aux | grep contextBroker)
   dMsg $ps
+
+  # Some times (specially when using remote DB) CB needs some time to connect DB and
+  # the test execution needs to a guard time. The righ value for CB_WAIT_AFTER_START
+  # needs to be find empirically (it seems that when DB is localhost this time is
+  # not needed at all)
+  #
+  # (A better solution will be to detect when CB is ready to start the test, but
+  # it can be complex. Note that and check above ensures that the port is ready, but
+  # it doesn't guarantee that DB connection is ok).
```

Comment:
nc on 27017 doesn't work, as we are not waiting for the database service to start (mongod will be typically be running for days or months), but waiting for CB to connect to DB.

Although restInit() cames after mongoInit() is seems that it doesn't suffice... Probably some is happening asynchronously. Maybe we should implement some mechanim in mongoInit() so the function doesn't return if the connection has not succesuffy stablished (but not in this PR but as a separate issue, by the moment CB_WAIT_AFTER_START would suffice).